assets/jump-logo-with-text-large.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="400" viewBox="0 0 377.42 122.03"><defs><style>.cls-1{fill:#231f20;}.cls-2,.cls-3,.cls-5,.cls-7,.cls-9{fill:none;}.cls-2,.cls-3{stroke:#231f20;stroke-linecap:round;stroke-linejoin:round;}.cls-2{stroke-width:6.63px;}.cls-3{stroke-width:6.63px;fill-rule:evenodd;}.cls-4{fill:#aa7dc0;}.cls-5{stroke:#945bb0;}.cls-5,.cls-7,.cls-9{stroke-miterlimit:10;stroke-width:3px;}.cls-6{fill:#6bab5b;}.cls-7{stroke:#3b972e;}.cls-8{fill:#d66661;}.cls-9{stroke:#c93d39;}</style></defs><title>Thin-Border-Logo-Text</title><g id="Page_1" data-name="Page 1"><g id="Layer_1-2" data-name="Layer 1"><polygon class="cls-1" points="313.71 117.21 326.3 117.21 326.3 50.53 311.9 50.53 291.01 79.2 270.12 50.53 255.62 50.53 255.62 117.21 268.21 117.21 268.21 68.72 289.48 97.78 291.96 97.78 313.71 68.81 313.71 117.21 313.71 117.21"/><path class="cls-1" d="M172.45,50.54H185.2V93.37c0,3.86.29,7.71-.65,11.5a19.26,19.26,0,0,1-7.72,11.33,19.75,19.75,0,0,1-9.07,3.42,20.85,20.85,0,0,1-22.67-16.11l12.38-3.16a8.45,8.45,0,0,0,5.17,6.09,8,8,0,0,0,3.67.45,7.09,7.09,0,0,0,3.2-1.16c2.77-1.93,2.94-5.34,2.94-8.34V50.54Zm26.46,0h12.52V74.35c0,4.61.31,7.84.95,9.63a8.46,8.46,0,0,0,3.06,4.19,8.76,8.76,0,0,0,5.2,1.51,9.15,9.15,0,0,0,5.24-1.47,8.68,8.68,0,0,0,3.2-4.34q.76-2.13.76-9.12V50.57h12.43V71.49c0,8.61-.69,14.52-2,17.69a19.21,19.21,0,0,1-7.34,8.9q-4.86,3.11-12.34,3.11-8.13,0-13.13-3.62a19.27,19.27,0,0,1-7.06-10.12c-.94-3-1.45-8.44-1.45-16.32V50.57Zm138.87,0h13.46q11,0,15.76,2a16,16,0,0,1,7.63,6.54,20.21,20.21,0,0,1,2.79,10.8,18.23,18.23,0,0,1-3.65,11.61A18.82,18.82,0,0,1,363.86,88q-3.66,1-13.37,1v28.26H337.78V50.54Zm12.71,26.08h4a22.42,22.42,0,0,0,6.63-.68,5.82,5.82,0,0,0,2.92-2.25,6.53,6.53,0,0,0,1.07-3.79,6.09,6.09,0,0,0-3-5.62C360.69,63.41,358,63,354.06,63h-3.57V76.62Z"/><line class="cls-2" x1="19.64" y1="67.28" x2="120.16" y2="28.42"/><polyline class="cls-3" points="3.32 118.72 89.75 32.29 118.72 3.32"/><path class="cls-4" d="M106.29,120.18a17.17,17.17,0,1,1,12.16-5A17.06,17.06,0,0,1,106.29,120.18Z"/><path class="cls-5" d="M106.29,120.18a17.17,17.17,0,1,1,12.16-5A17.06,17.06,0,0,1,106.29,120.18Z"/><path class="cls-6" d="M106.29,79.33a17.19,17.19,0,1,1,12.16-5A17.07,17.07,0,0,1,106.29,79.33Z"/><path class="cls-7" d="M106.29,79.33a17.19,17.19,0,1,1,12.16-5A17.07,17.07,0,0,1,106.29,79.33Z"/><path class="cls-8" d="M65.28,120.18a17.19,17.19,0,1,1,12.15-5A17,17,0,0,1,65.28,120.18Z"/><path class="cls-9" d="M65.28,120.18a17.19,17.19,0,1,1,12.15-5A17,17,0,0,1,65.28,120.18Z"/></g></g></svg>
```

Comment:
I'm wondering if we need two copies of the logo checked in. It's an SVG so shouldn't be hard to rescale inline.